packages/vx-axis/src/utils/center.js

```diff
@@ -1,7 +1,7 @@
 export default function center(scale) {
-  var offset = scale.bandwidth() / 2;
+  let offset = scale.bandwidth() / 2;
```

Comment:
I think this actually still needs to be `var` because of the closure, did you test the gallery locally to see if there were no errors?